src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkInformationException.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```

Comment:
I'm not sure why GitHub isn't showing the newest version of this file in the PR: https://github.com/mellinoe/corefx/blob/ni-xplat/src/Common/src/System/Net/NetworkInformation/NetworkInformationException.cs